documentation/element-api/tutorial-dynamic-styling.asciidoc

```diff
@@ -16,15 +24,18 @@ To change the look and feel of your application, you can style it by using dynam
 }
 ----
 
+*Example*: Using the `getClassList()` method to dynamically modify the class names of an element.
+
 [source,java]
 ----
 button.setText("Change to blue");
 button.addEventListener("click",
     e -> button.getClassList().add("blue"));
 ----
 
-In this case you are limited to the styles you have defined in advance. To get really dynamic styles, you can use the inline styling API `getStyle()`:
+* Using  `getClassList()` limits you to styles that have been defined in advance. 
```

Comment:
```suggestion
```

One can inject CSS dynamically as well, so it doesn't have to be "in advance". All in all, this is rather obsolete sentence.